### DISCUSSION TITLE
how to find a word and translate it

### DISCUSSION THREAD

**Discussion by aucker** [gdh1995/vimium-c#1015] [Q&A] [answered]
I know how to use `visual mode` to find a word, but after finding the word,  is there any way to trigger a translation tool extension like `google dictionary` or `saladict` to translate this word?

just like the screenshot:
![image](https://github.com/gdh1995/vimium-c/assets/41871212/34e22fd8-ace5-4196-a6c0-c4d59853872a)
I don't want to use the mouse and right click and choose saladict and then translate.😢

**Comment by aucker** [comment]
Just like the VS Code Vim, there is a hover preview to show the code definition, I wonder is that any way for the extension to show the translation for vimium-c. 🫡

**Comment by gdh1995** [comment, marked as answer]
This usage has been answered in https://github.com/gdh1995/vimium-c/issues/247 and I wish it still works now - if not please fell free to open a new bug report or add a comment here.

**Reply by aucker** [reply]
Thanks a lot, this is exactly what I want, it works with Google Dictionary and I can use it directly now.🫡
But it's still a pity there is no right-click (macOS trackpad two-finger tap is also the right-click?) browser API or I can also use Apple Dictionary hahaha. Thanks >_
